libafl/src/fuzzer/mod.rs

```diff
@@ -447,7 +461,13 @@ where
     OT: ObserversTuple<CS::State> + Serialize + DeserializeOwned,
     F: Feedback<CS::State>,
     OF: Feedback<CS::State>,
-    CS::State: HasCorpus + HasSolutions + HasClientPerfMonitor + HasExecutions,
+    CS::State: HasCorpus
```

Comment:
you don't need these type constraints anymore
then fuzzer/mod.rs looks good.

Reply:
Okey